src/ogd/games/BLOOM/features/BloomAlertCount.py

```diff
@@ -0,0 +1,39 @@
+# import libraries
+from typing import Any, Dict, List, Optional
+from ogd.core.generators.Generator import GeneratorParameters
+from ogd.core.generators.extractors.Feature import Feature
+from ogd.core.models.Event import Event
+from ogd.core.models.enums.ExtractionMode import ExtractionMode
+from ogd.core.models.FeatureData import FeatureData
+
+class BloomAlertCount(Feature):
+    def __init__(self, params: GeneratorParameters):
+        super().__init__(params=params)
+        self.bloom_alert_counts: Dict[str, int] = {}
+
+    # *** IMPLEMENT ABSTRACT FUNCTIONS ***
+    @classmethod
+    def _eventFilter(cls, mode: ExtractionMode) -> List[str]:
+        return ["algae_growth_begin", "algae_growth_end"]
```

Comment:
We want to use `bloom_alert_displayed`, not `algae_growth_begin` and `algae_growth_end`.

Reply:
resloved